### ISSUE TITLE
Typescript declaration is wrong

### ISSUE THREAD

**Issue by egorovsa** [yisheng90/react-loading#8]
Hi. first of all you have typings errors

```
  height?: stting | number // should be string
  width?: stting | number // should be string
```

then typescript in not recognise you d.ts file because it's not in the root folder, moreover you should declare your d.ts in the module's package.json

so
1) rename your Selector.d.ts  to index.d.ts
2) move it to the module root
3) add new prop to your package.json
`  "typings": "index.d.ts",`

and finally your index.d.ts file should look smth like that

```
interface SkeletonProps {
    rows?: number
    circle?: boolean
    color?: string
    translucent?: boolean
    height?: string | number
    width?: string | number
}

declare const Skeleton: (props: SkeletonProps) => any;
export default Skeleton;
```

please fix it

**Comment by yisheng90** [comment]
@egorovsa,

Thank you for sharing how the typing declaration would works. I was experimenting it before. 

I have made the changes and published the fix at version 1.4.3. 

Once again, thank you for sharing.

Regards,
Yi Sheng

**Comment by egorovsa** [comment]
Woks fine now, Thx!
